VideoLocker/res/values/text_styles.xml

```diff
@@ -103,19 +103,27 @@
     <style name="bold_white_text" parent="@style/bold_text">
         <item name="android:textColor">@android:color/white</item>
     </style>
+
+    <style name="regular_black_text" parent="@style/regular_text">

```

Comment:
`black` is defined as `#111010` in the [color palette](http://ux.edx.org/elements/colors/). `#000000` corresponds to `black_t` which I've wrongly used here. Let me fix that.

>It could also be something more semantic like "body text"?

Actually these are some base styles which only define the *color* and *text-style* of a particular text and are usually being extended by other more semantic styles like `course_card_title` and `course_card_course_run`